### PR TITLE
Upgrade `finch-tensor`

### PR DESCRIPTION
**Pull request by mtsokol**

### CODE REVIEW

File: examples/sddmm_example.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import scipy.sparse as sps
 
 LEN = 10000
-DENSITY = 0.0001
+DENSITY = 0.00001
```
Comment (hameerabbasi) [inline examples/sddmm_example.py:9]: Why do we need this change?

**Review by hameerabbasi — COMMENTED** [review]
A question.

File: examples/sddmm_example.py (hunk 1 above)
Comment (mtsokol) [reply examples/sddmm_example.py:9]: This example runs a bit faster now - SDDMM takes the longest time out of all examples. It also shows Finch speedup even better.

**Review by hameerabbasi — APPROVED** [review]

**Comment by mtsokol** [comment]
Can we also make a new alpha release?

**Comment by hameerabbasi** [comment]
Well, we don't need one for this do we? It's bumping the lower bound on a dependency, but one could just do `pip install -U finch-tensor`.

**Comment by mtsokol** [comment]
> Well, we don't need one for this do we? It's bumping the lower bound on a dependency, but one could just do `pip install -U finch-tensor`.

Last release was on 13.05, we still don't have https://github.com/pydata/sparse/pull/678 released in another alpha. I think we could have a new one.

**Comment by hameerabbasi** [comment]
> Last release was on 13.05, we still don't have #678 API changes released in another alpha. I think we could have a new one.

Done! https://pypi.org/project/sparse/0.16.0a7/

**Comment by mtsokol** [comment]
Thank you!

**Pull request merged**